.babelrc

```diff
@@ -1,6 +1,9 @@
 {
-  "stage": 1,
-  "optional": ["runtime"],
-  "loose": ["all"],
-  "plugins": ["dev-expression"]
```

Comment:
Ah interesting! I didn't realize this did more than replace `__DEV__`, looks pretty useful!